### PR TITLE
Adding a -long test parameters to run long-running tests

### PR DESCRIPTION
**Pull request by ChristianMct**
As in tite.

### CODE REVIEW

File: dckks/dckks_test.go [hunk 1]
```diff
@@ -58,12 +59,12 @@ func TestDCKKS(t *testing.T) {
...
 	var testCtx = new(testContext)
 
-	var defaultParams []*ckks.Parameters
-
+	var defaultParams = ckks.DefaultParams[ckks.PN12QP109 : ckks.PN12QP109+3] // the default test runs for ring degree N=2^12, 2^13, 2^14
 	if testing.Short() {
-		defaultParams = ckks.DefaultParams[ckks.PN12QP109 : ckks.PN12QP109+3]
-	} else {
-		defaultParams = ckks.DefaultParams
+		defaultParams = ckks.DefaultParams[ckks.PN12QP109 : ckks.PN12QP109+2] // the short test runs for ring degree N=2^12, 2^13, 2^14, 2^15
```
Comment (Pro7ech) [inline dckks/dckks_test.go:64]: This will go up to 2^{13} only

File: dckks/dckks_test.go [hunk 2]
```diff
@@ -58,12 +59,12 @@ func TestDCKKS(t *testing.T) {
 	var err error
 	var testCtx = new(testContext)
 
-	var defaultParams []*ckks.Parameters
-
+	var defaultParams = ckks.DefaultParams[ckks.PN12QP109 : ckks.PN12QP109+3] // the default test runs for ring degree N=2^12, 2^13, 2^14
```
Comment (Pro7ech) [inline dckks/dckks_test.go:62]: Set to same as the other schemes

**Review by Pro7ech — CHANGES_REQUESTED** [review]

File: dckks/dckks_test.go (hunk 2 above)
Comment (ChristianMct) [reply dckks/dckks_test.go]: fixed

File: dckks/dckks_test.go (hunk 1 above)
Comment (ChristianMct) [reply dckks/dckks_test.go]: Good catch, fixed

**Review by Pro7ech — APPROVED** [review]

**Pull request merged**